app/javascript/components/rules/RuleNavigator.vue

```diff
@@ -34,37 +34,11 @@
     </p>
```

Comment:
the `sortById` method should be updated or replaced to avoid this kind of confusing sorting 
![Screen Shot 2021-10-01 at 9 23 10 AM](https://user-images.githubusercontent.com/24628687/135645927-f49a4b9f-4ca6-414b-a0a1-470f323dd952.png)
on